packages/blockchain-wallet-v4-frontend/src/data/components/nfts/sagas.ts

```diff
@@ -229,9 +229,9 @@ export default ({ api }: { api: APIType }) => {
       yield put(A.fetchFeesLoading())
       const signer: Signer = yield call(getEthSigner)
       let fees: GasDataI
+      // TODO: DONT DEFAULT TO 1 WEEK
+      const expirationTime = moment().add(14, 'day').unix()
       if (action.payload.operation === GasCalculationOperations.Buy) {
-        // TODO: DONT DEFAULT TO 1 WEEK
-        const expirationTime = moment().add(7, 'day').unix()
```

Comment:
Ideally a user is able to select this via some widget.

Reply:
Yeah, working on that right now.  https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/4080/commits/a75d88aa6d919c505e9a1f4e4e3b1bad5dc5a90c